src/captured_trace.js

```diff
@@ -452,7 +452,12 @@ var fireGlobalEvent = (function() {
         } catch (e) {}
         if (!customEventWorks) {
             try {
-                var event = document.createEvent("CustomEvent");
+                var event;
+                if (document.createEventObject) {
```

Comment:
possibly this would be better split out as a utility function since its used twice? the comment would go nicely there.